@uportal/esco-content-menu/src/components/PortletCard.vue

```diff
@@ -218,6 +218,51 @@ export default {
     font-size: 14px;
   }
 
+  &.custom-card {
+    padding: 5px;
+    padding: var(--content-gridcard-padding, 5px);
+    border: none;
+    border: var(--content-gridcard-border, none);
+    background-color: white;
+    background-color: var(--content-gridcard-bg-color, white);
+    border-radius: 5px;
+    border-radius: var(--content-gridcard-border-radius, 5px);
+    box-shadow: none;
+    box-shadow: var(--content-gridcard-shadow, none);
+    transition: box-shadow 0.25s;
+    width: $PortletCardSizeCustomWidth;
+    width: var(--content-gridcard-size-w, $PortletCardSizeCustomWidth);
+    height: $PortletCardSizeCustomHeight;
+    height: var(--content-gridcard-size-h, $PortletCardSizeCustomHeight);
+
+    &:hover {
+      cursor: pointer;
```

Comment:
`cursor: pointer` is already defined so is useless to define it again

Reply:
removed